kraken/litmus/common_litmus.py

```diff
@@ -2,6 +2,52 @@
 import logging
 import time
 import sys
+import requests
+import yaml
+import kraken.cerberus.setup as cerberus
+
+
+# Inject litmus scenarios defined in the config
```

Comment:
Very small nit pick, all other scenario types you used "run" as the function name and here you used scenarios. Might be good to keep it consistent across all scenario types to use "run" as the main function

Reply:
Absolutely, fixed it.